cime_config/buildnml

```diff
@@ -144,29 +145,39 @@ def buildnml(case, caseroot, compname):
                 or clm_usrdat_name is "NEON.PRISM"
             ):
                 logger.warning(
-                    "WARNING: Do you have approriprate initial conditions for this simulation?"
-                    + " Check that the finidat file used in the lnd_in namelist is apprporiately spunup for your case"
+                    "WARNING: Do you have appropriate initial conditions for this simulation?"
```

Comment:
Can this be more verbose? 
Something like: "DGL_USE_ANTARCTICA is not compatible with CLM_FORCE_COLDSTART=True and initial conditions from non-transient NEON runs or NEON.PRISM runs."?

Reply:
The lines you are referring to  are not related to GLC (and in the meantime the xml variable has been moved from DGLC_USE_ANTARCTICA to GLC_USE_ANTARCTICA).  Its simply NEON related and the original comment.  I'm not sure I totally understand the use case they are referring to. And it relates to user generated data. I'm not sure what the exact new wording should be. I'm happy to talk about this.

Reply:
Oh, ok. Then just leave it. ESCOMP should deal with this.